.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: python
 python:
   - "2.7"
-sudo: false
+before_install:
+    - pip install git+https://github.com/Erf-/wifi.git@connection-status#egg=wifi
+    - sudo chmod a+r+w /etc/network/interfaces
```

Comment:
Tests should not touch this file for real, is there some place we are not mocking?